src/components/attendee/Popover.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import ReactDOM from "react-dom";
+import { usePopper } from "react-popper";
+
+import { POPOVER_CONTAINER_ID } from "settings";
+
+export interface PopoverProps {
+  referenceElement?: Element | null;
+}
+
+export const Popover: React.FC<PopoverProps> = ({
+  children,
+  referenceElement,
+}) => {
+  const popoverContainerElement = document.querySelector(
+    `#${POPOVER_CONTAINER_ID}`
+  );
+
+  const [popperElement, setPopperElement] = useState<HTMLElement | null>(null);
+  const { styles: popperStyles, attributes: popperAttributes } = usePopper(
+    referenceElement,
+    popperElement
+  );
+
+  return (
+    popoverContainerElement &&
+    ReactDOM.createPortal(
```

Comment:
Maybe we can put a comment or two of why `createPortal` is needed instead of a common JSX